cmake/macros/macro_deal_ii_invoke_autopilot.cmake

```diff
@@ -141,13 +141,17 @@ MACRO(DEAL_II_INVOKE_AUTOPILOT)
   # Define custom targets to easily switch the build type:
   ADD_CUSTOM_TARGET(debug
     COMMAND ${CMAKE_COMMAND} -DCMAKE_BUILD_TYPE=Debug ${CMAKE_SOURCE_DIR}
-    COMMAND ${CMAKE_COMMAND} --build ${CMAKE_BINARY_DIR} --target all
+    COMMAND ${CMAKE_COMMAND} -E echo "***"
+    COMMAND ${CMAKE_COMMAND} -E echo "*** Switched to Release mode. Now recompile with: ${_make_command}"
```

Comment:
I needed to add "VERBATIM" to make that work. Did you test this?

Reply:
Wow. This indeed needs the `VERBATIM` keyword. The answer is yes I tested the functionality beforehand but not with the final three line statement but only a single-line one.